flixel/FlxSprite.hx

```diff
@@ -630,6 +631,9 @@ class FlxSprite extends FlxObject
 	{
 		if (_frame == null)
 			loadGraphic("flixel/images/logo/default.png");
+		
+	//	if (_frame == null)
```

Comment:
Better to remove than comment out, unless there is good reason to believe it will be useful in the future.